modules/340-monitoring-kubernetes/templates/node-exporter/daemonset.yaml

```diff
@@ -67,6 +67,7 @@ spec:
       serviceAccountName: node-exporter
       {{- include "helm_lib_priority_class" (tuple . "system-node-critical") | nindent 6 }}
       {{- include "helm_lib_tolerations" (tuple . "any-node") | nindent 6 }}
+#TODO: we need to run it from the deckhouse user, but it's not clear it how
```

Comment:
```suggestion
```
Let's copy settings from the [official damonset](https://github.com/prometheus-operator/kube-prometheus/blob/main/manifests/nodeExporter-daemonset.yaml).

1. SYS_TIME capability should be added for node-exporter.
2. `mountPropagation: HostToContainer` option should be added to the `/host/root` volume mount.
3. Default pod-level security context should be:
```
securityContext:
  runAsNonRoot: true
  runAsUser: 65535
``` 
(because the root is not required for kube-rbac-proxy).